src/main/java/ca/serum390/godzilla/api/filters/ReactRouterForwardFilter.java

```diff
@@ -43,13 +39,7 @@ private boolean matchesReactRouterRoute(ServerWebExchange exchange) {
      * @return
      */
     private Mono<Void> forwardToReactRouter(ServerWebExchange exchange, WebFilterChain chain) {
-        return chain.filter(
-                exchange.mutate()
-                        .request(
-                            exchange.getRequest()
-                                    .mutate()
-                                    .path("/index.html")
-                                    .build()
-                        ).build());
+        return chain
```

Comment:
Also reformat this line so that it is readable:

```java
        return chain.filter(
                exchange.mutate()
                        .request(
                            exchange.getRequest()
                                    .mutate()
                                    .path("/index.html")
                                    .build()
                        ).build());
```